### PR TITLE
Finding home for lost Seattle Data

### PR DESCRIPTION
**Pull request by dostran4**
From Issue #749, I found a video source for one complaint code. I also researched and found a news article for another complaint code, but couldn't locate the pictures mentioned in the article.

### CODE REVIEW

File: reports/Washington.md [hunk 1]
```diff
@@ -34,7 +34,21 @@ id: wa-seattle-1
 **Links**
 
 * https://mobile.twitter.com/chadloder/status/1267011092045115392
-* 2020OPA-0324 - May 30 - "Placing the knee on the neck area of two people who had been arrested" (possibly)
+* Possibly related to OPD Case 2020OPA-0324 - "Placing the knee on the neck area of two people who had been arrested"
```
Comment (emarcey) [inline reports/Washington.md:37]: This should go in the description, rather than the link.

File: reports/Washington.md [hunk 2]
```diff
@@ -34,7 +34,21 @@ id: wa-seattle-1
...
+
+According to a news report, a oman at the Saturday, June 1st protest in downtown Seattle sustained severe injuries from sharpnel that pierced her thumb and forefinger.
+
+tags: flashbang
+
+id: wa-seattle-55
+
+**Links**
+
+* https://www.seattletimes.com/seattle-news/seattle-police-continue-to-use-flash-bang-grenades-during-protests-despite-recommendations/
```
Comment (emarcey) [inline reports/Washington.md:50]: tbh I don't think this is solid enough to call an incident. One person recalling that she saw an incident and telling a reporter later is not quite concrete.

File: reports/Washington.md [hunk 3]
```diff
@@ -165,6 +179,20 @@ id: wa-seattle-10
 * Related to OPD Case 2020OPA-0333 "A flashbang striking a reporter"
 
 
+### Police injure sleeping homeless man with flashbang | June 1st
+
+Police flashbang hits a homeless man sleeping in a park.
+
+tags: flashbang, homeless
```
Comment (emarcey) [inline reports/Washington.md:186]: please add the bystander tag as well

File: reports/Washington.md [hunk 4]
```diff
@@ -165,6 +179,20 @@ id: wa-seattle-10
...
+
+id: wa-seattle-54
+
+**Links**
+
+* https://www.facebook.com/kg.farmer/videos/278656296661372
+* Related to OPD Case 2020OPA-0335 "Homeless man hit by an SPD flashbang while sleeping"
```
Comment (emarcey) [inline reports/Washington.md:193]: I think all of these should go into description rather than Links because they're not links

**Comment by dostran4** [comment]
> see comments

Should be fixed now.

**Pull request merged**